.github/workflows/gsas2full.yml

```diff
@@ -68,14 +59,21 @@ jobs:
           cd install
           CONDA_SOLVER=classic constructor g2full   # can't use mamba in constructor 3.7
 
-      - name: Test the g2full installer
-        shell: bash -l {0}
-        run: |
-          if [ "$RUNNER_OS" != "Windows" ]; then
-             ls -lt install/*.sh
-             #rm -rf ~/build
-             #bash `ls install/*.sh` -b -p ~/testinstall
-          fi
+# Now have a gsas2full-####-<platform>.sh or gsas2full-####-<platform>.exe file in
+# install. That file should get uploaded to the Assets area here:
+#    https://github.com/AdvancedPhotonSource/GSAS-II-buildtools/releases/latest
+# It should also be copied to replace file gsas2full-Latest-<platform>.*
```

Comment:
I've used the following GitHub action to add files to a GitHub release page: https://github.com/softprops/action-gh-release